format/index.md

```diff
@@ -1453,6 +1470,9 @@ constraints (such as a uniqueness constraint on a property other than `id`).
 A server **MUST** return `409 Conflict` when processing a `PATCH` request in
 which the resource object's `type` and `id` do not match the server's endpoint.
 
+A server **MUST** return `409 Conflict` when processing a `PATCH` request in
```

Comment:
I think this is too prescriptive. A server may be able to resolve a conflict and accept it.